addons/survey/views/survey_question_views.xml

```diff
@@ -148,7 +155,7 @@
                                         <field name="sequence" widget="handle"/>
                                         <field name="value" string="Choices"/>
                                         <field name="value_image" options="{'accepted_file_extensions': 'image/*'}"
-                                            attrs="{'column_invisible': ['|', ('parent.allow_value_image', '=', False), ('parent.question_type', '=', 'matrix')]}"/>
```

Comment:
Not sure why this change?

Reply:
Because of this:

LNA CHANGE REQUEST 09/09/2021:
    Let's change a bit the "image" implementation:
    Instead of having it as an option here: https://tinyurl.com/yfaxoty3
    We always display the image field https://tinyurl.com/ygbvo75k
=> And we adapt the display of the survey as soon as an image is uploaded for this question

Reply:
Ok, but what about "allow_value_image" then?
Is it even displayed anywhere anymore?
If we don't need it, we need to clean it and remove it.

Reply:
I skipped it by accident. It is now done.